dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -396,13 +402,10 @@
     <Style x:Key="NavigationViewOverflowButtonStyleWhenPaneOnTop" TargetType="Button">
```

Comment:
Ideally, this style and the `NavigationViewOverflowButtonNoLabelStyleWhenPaneOnTop` style directly below, now that they will become identical, could be moved into a single style called `TopNavigationViewOverflowButtonStyle` (which would also match the common naming scheme used in this file). 

This could happen alongside the removal of the visual states for the OverflowLabelMode (or event the removoal of the NavigationView.OverflowLabelMode) as part of major WinUI release, such as WinUI 4.0.

Should I add a comment here that these styles could be merged into a single style in the future?

Reply:
I think that makes sense. @ryandemopoulos @MikeHillberg and @stmoy do we have a place where we are collecting a list of changes we want to delay to WinUI 4.0 for semver reasons?

Reply:
Not that I am aware of. I would recommend filing these as issues in GitHub and mark them with a new label for tracking.

Reply:
Ok, will create a new tracking issue for this.